_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,145 @@
+---
+name: ARIA required context role
+description: | 
+  This rule checks that a role does not exist outside of its required context roles
+
+success_criterion:
+- 1.3.1 # Info and Relationships
+
+test_aspects: 
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role) that has [required context roles](https://www.w3.org/TR/wai-aria-1.1/#scope) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria).
+
+### Expectation
+
+Each target element is [owned](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element) by an element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has a [semantic role](#semantic-role) matching one of the [required context roles](https://www.w3.org/TR/wai-aria-1.1/#scope) listed for that role in [WAI-ARIA](https://www.w3.org/TR/wai-aria).
+
+## Assumptions
+
+_There are currently no assumptions_
+
+## Accessibility Support
+
+When a required owned element is not a child, but rather a descendant, certain AT have issues recognizing the owned element.
+
+## Background
+
+- [Required Context Role](https://www.w3.org/TR/wai-aria-1.1/#scope)
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+Element with role ´listitem´ is contained within its required context role ´list´, expressed as an explicit role.
+
+```html
+<div role="list">
+    <div role="listitem"></div>
+</div>
+```
+
+#### Passed example 2
+
+Multiple levels of required context roles.
+
+```html
+<div role="table">
+    <div role="row">
+        <div role="cell"></div>
+    </div>
+</div>
+</div>
+```
+
+#### Passed example 3
+
+Element with role ´listitem´ is contained within its required context role ´list´, through the implicit role of ´ul´.
+
+```html
+<ul>
+    <div role="listitem"></div>
+</ul>
+```
+
+#### Passed example 4
+
+Element contained within its required context role even though it is not a direct child of the context role.
+
+```html
+<div role="list">
+    <div>
+         <div>
+             <div>
+                <div role="listitem"></div>
```

Comment:
This does not work, and should not pass this rule.

Reply:
I works perfectly fine in VoiceOver.

Reply:
If it does not work, it is a question of lacking accessibility support, since it follows the spec and works in at least some technologies.

Reply:
If I understand correctly, this example does follow the WAI-ARIA spec and some ATs support it. Therefore it should pass the rule, with an accessibility support note as defined above. Ideally we would have a parallel "good practice" rule to catch these situations and issue a warning (as opposed to a conformance fail).

Reply:
@WilcoFiers, do you agree that this is a pass with the new definition of "owner element"?

Reply:
My 2 cents: according to the new definition it is a pass